README.md

```diff
@@ -13,55 +13,122 @@ Quickstart
 ----------
 
 - Install Nix
-- Enable Flakes support
+- (Optional) Enable Flakes support
 - (Optional) Install direnv
+- Clone this repository
 
-If using direnv: Allow this repository. Else: Enter the Nix Shell using `nix develop`.
+If using direnv: Allow this repository. Else: Enter the Nix Shell using `nix develop`, unless you are not using flakes, then
+`nix-shell shell.nix`
 
-You will see a menu with options available for Rodeo.
+Detailed instructions
+----------------------
+The first thing to do is to install Nix. This is a package manager. It will allow you to download
+dependencies in an isolated way.
 
-Installation
-------------
+To install Nix in Linux run:
+
+    sh <(curl -L https://nixos.org/nix/install) --daemon
+
+In macOS (Darwin) don't use `--daemon`:
+
+    sh <(curl -L https://nixos.org/nix/install)
+
+For more information about Nix installation you can check their [documentation](https://nixos.org/manual/nix/stable/installation/installation.html).
+
+If you are using NixOS, you already have Nix installed.
+
+### Flakes support
+
+Flakes is a feature in the Nix package manager. It allows us to create
+isolated environments.
+
+To enable Flakes:
+
+- In Linux/macOS: `~/.config/nix/ && echo "experimental-features = nix-command flakes" >> ~/.config/nix/`
```

Comment:
I think this command is incomplete?

Usually in a fresh install there is no `.config` folder I think, so we might need to do `mkdir -p ~/.config/nix/`